fhir-client/src/main/java/com/ibm/fhir/client/FHIRClient.java

```diff
@@ -344,7 +344,10 @@
      * @return a FHIRResponse that contains the results of the 'search-all' operation
      * @throws Exception
      */
-    FHIRResponse searchAll(FHIRParameters parameters, FHIRRequestHeader... headers) throws Exception;
+    FHIRResponse searchAllGet(FHIRParameters parameters, FHIRRequestHeader... headers) throws Exception;
+    
+    
+    FHIRResponse searchAllPost(FHIRParameters parameters, FHIRRequestHeader... headers) throws Exception;
```

Comment:
I think it's worth documenting the reason that there are two interfaces.  some javadocs might help explain.

Reply:
yeah, this is align with FHIR spec. will add

Reply:
done